theseus/geometry/se2.py

```diff
@@ -17,6 +17,8 @@
 # If data is passed, must be x, y, cos, sin
 # If x_y_theta is passed, must be tensor with shape batch_size x 3
 class SE2(LieGroup):
+    SE2_EPS = 5e-7
```

Comment:
I also think it makes sense to move this into constants.

Reply:
@fantaosha was this address in a later PR?

Reply:
I think it's better to address this in a separate PR as part of the solution for #61, given that EPS needs to be different for each dtype, which is object dependent. I have a proposal for how to do this part, and I was waiting for SO3 to be merged into main. I'll try to open a PR for this today to discuss.

Reply:
I'm still wondering if we should let these constants go to `constants.py`. Since `SO3` and `SE3` have lots of member functions using numerous different `EPS` to handle cases near to `0` and `pi`, I'm afraid there will be too many in `constants.py`.

Reply:
> I think it's better to address this in a separate PR as part of the solution for #61, given that EPS needs to be different for each dtype, which is object dependent. I have a proposal for how to do this part, and I was waiting for SO3 to be merged into main. I'll try to open a PR for this today to discuss.

Oh, sorry. I just created #70. Feel free to close #70 if you want a new PR.

Reply:
Should we wait until SE3 is finished to assess which (if any) of these constants would make sense to move?

Reply:
> Should we wait until SE3 is finished to assess which (if any) of these constants would make sense to move?

@fantaosha Seems fine to keep #70 for now until we decide what to do about this.